payment_term_type/README.rst

```diff
@@ -1,8 +1,65 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==================
 Payment Terms Type
 ==================
 
-This module allow to show in payment terms the payment type depending of the lines to compute
+This module allow to show in payment terms the payment type depending of the
+lines to compute
+In Settings> Account you can find a selection field payment_type that can be
```

Comment:
When you say

> In Settings> Account you can find a selection field payment_type 

I think will better to refer the exact path of the menu and mention the human readable name of the field, for example: 

> In the`Settings > Configuration > Accounting` wizard you can find the the **Payment Term Type** (`payment_type)` option